### ISSUE TITLE
how to override the default definition

### ISSUE THREAD

**Issue by smancang** [microsoft/vscode#76231]
I develop an VSCode extension using `vscode.languages.registerDefinitionProvider` method to implement the function 'Go to Definition', because my project uses Webpack alias，the VSCode default 'go to definition' doesn't work correctly. When I use the extension, VSCode shows 2 definitions for me to choose, of which one is the VSCode default feature that doesn't work right.

I don't want the default, not correct 'go to definition' shows. I don't want there is one more operation for user to choose which definition to go.Just go where I want it to go.

In other words, I need to override the default feature 'go to definition', not to cooperate with.

**Comment by jrieken** [comment]
> In other words, I need to override the default feature 'go to definition', not to cooperate with.

The extension system is designed to be cooperative and so that a single extension cannot mute results of other extensions. We have no plans of changing that. However, if you think that the TypeScript/JavaScript extension behaves incorrect for a given scenario that please file a new, separate issue with steps to reproduce. Thanks

**Comment by smancang** [comment]
There's someone else having the same problem [#52996](https://github.com/Microsoft/vscode/issues/52996).I don't know why it is closed.Haven't you use webpack's alias configuration? 
If you config the webpack.config.js like this

module.export = {
// ...
resolve: {
      alias: {
        components: resolve('./src/components/'),
      },
    },
}
//...

then you can use `import { A } from 'components'` instead of `import { A } from './src/components'`.
At this moment, if you press 'ctrl' and click  the word "A" in the statement `import { A } from 'component'`, it won't go to the right definition.
